server/src/main/java/io/druid/segment/realtime/plumber/NoopSegmentHandoffNotifierFactory.java

```diff
@@ -0,0 +1,34 @@
+package io.druid.segment.realtime.plumber;
+
+import io.druid.java.util.common.logger.Logger;
+import io.druid.query.SegmentDescriptor;
+
+import java.util.concurrent.Executor;
+
+public class NoopSegmentHandoffNotifierFactory implements SegmentHandoffNotifierFactory
+{
+  private static final Logger log = new Logger(NoopSegmentHandoffNotifierFactory.class);
+
+  @Override
+  public SegmentHandoffNotifier createSegmentHandoffNotifier(String dataSource)
```

Comment:
`dataSource` is not used in the returned `SegmentHandoffNotifier`. Not sure if it is as intended or not. If it is not, maybe cache a SegmentHandoffNotifier in a static field?